docs/pages_ru/usage/deploy/debug_templates.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Отладка шаблонов
+permalink: usage/deploy/debug_templates.html
+---
+
+## Отладка шаблонов манифестов
+
+{% include pages/ru/debugging_templates.md.liquid %}
+
+### -s, --show-only=[]
+
+Показывать только манифесты, отрендеренные из указанных шаблонов.
+
+Флаг позволяет ограничить вывод командой werf render (или другими командами, использующими рендеринг Helm-шаблонов), чтобы отображались только манифесты, сгенерированные из определённых файлов шаблонов.
+
+```bash
+werf render -s .helm/charts/frontend/templates/deployment.yaml
+```
+
+```yaml
+---
+# Source: demo-app/charts/frontend/templates/deployment.yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  annotations:
+    meta.helm.sh/release-name: demo-app
+    meta.helm.sh/release-namespace: demo-app
+    project.werf.io/env: ""
+    project.werf.io/name: demo-app
+    werf.io/version: v2.36.4
+    werf.io/weight: "30"
+  labels:
+    app.kubernetes.io/managed-by: Helm
+  name: frontend
+spec:
+  selector:
+    matchLabels:
+      app: frontend
+  template:
+    metadata:
+      labels:
+        app: frontend
+    spec:
+      containers:
+      - image: REPO:TAG
+        name: frontend
+        ports:
+        - containerPort: 80
+        resources:
+          limits:
+            cpu: 50m
+            memory: 48Mi
+          requests:
+            cpu: 50m
+            memory: 48Mi
+        volumeMounts:
+        - mountPath: /etc/nginx/nginx.conf
+          name: config
+          subPath: nginx.conf
+      volumes:
+      - configMap:
+          name: frontend-config
+        name: config
+```
+
+### --debug-templates
+
+Флаг `--debug-templates` включает расширенный режим отладки Go-шаблонов в werf. Это позволяет писать отладочную информацию в лог без влияния на результат выполнения шаблонов.
+
+### Как включить
+
+Флаг можно активировать через CLI или переменной окружения:
+
+```bash
+werf render --debug-templates
+```
+
+или
+
+```bash
+export WERF_DEBUG_TEMPLATES=true
+werf render
+```
```

Comment:
I guess they can figure it out on their own, nothing special here. Remove section.